bobber/lib/exit_codes.py

```diff
@@ -0,0 +1,6 @@
+# SPDX-License-Identifier: MIT
+# This file contains a list of exit codes for debugging
+SUCCESS = 0  # Successful termination
+BASELINE_FAILURE = 10  # Performance did not meet criteria
```

Comment:
Should failure conditions be negative, rather than positive?

Reply:
Good idea, I can update that.